connectors/src/api/get_connector_permissions.ts

```diff
@@ -55,11 +55,12 @@ const _getConnectorPermissions = async (
   const connectorPermissionRetriever =
     RETRIEVE_CONNECTOR_PERMISSIONS_BY_TYPE[connector.type];
 
-  const pRes = await connectorPermissionRetriever(
-    connector.id,
+  const pRes = await connectorPermissionRetriever({
+    connectorId: connector.id,
     parentInternalId,
-    filterPermission
-  );
+    filterPermission,
+    retrieveAncestors: req.query.retrieveAncestors === "true",
```

Comment:
Let's use `parents` in place of `ancestors` everywhere :pray:

Reply:
done